packages/markdown-editor/package.json

```diff
@@ -44,7 +44,7 @@
     "ts-loader": "^9.2.6",
     "ts-node": "^10.4.0",
     "typescript": "4.4.3",
-    "webpack": "^5.64.1",
```

Comment:
Just curious, was there are reason to downgrade webpack?

Reply:
Actually webpack was upgraded:
![image](https://github.com/Joystream/pioneer/assets/6571453/18cd5c7d-8dd7-4468-9e44-d3a0f808dc6c)

It's just that there was initially duplicated versions of webpack after upgrading storybook. `yarn dedupe` would have taken care of that but IMO it's just as good to just force the major version of webpack (unless we realized later that some particular minor version breaks the build).